### PR TITLE
Only log flush error when there is one.

### PR DESCRIPTION
**Pull request by jeschkies**

### CODE REVIEW

**Review by MichelHollands — APPROVED** [review]
LGTM

**Review by dannykopping — CHANGES_REQUESTED** [review]
This error message won't make much sense to the user; can we make it more clear what is the problem exactly?

File: pkg/util/log/log.go [hunk 1]
```diff
@@ -170,7 +170,9 @@ func CheckFatal(location string, err error, logger log.Logger) {
 
 	logger.Log("err", errStr)
 	err = Flush()
-	fmt.Fprintln(os.Stderr, "Could not flush logger", err)
+	if err != nil {
```
Comment (owen-d) [inline pkg/util/log/log.go:173]: This looks unnecessary as the first line of this `CheckFatal` function is
```
	if err == nil {
		return
	}
```

**Comment by owen-d** [comment]
Closing, please reopen if I've missed something.

**Pull request closed without merging**